ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-stable-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -46,6 +46,11 @@ releases:
       architecture: multi
       channel: candidate
       version: "4.17"
+  source:
```

Comment:
Shall we rename a bit to indicate its usage purpose, for example, something like `preupg-test` ?

Reply:
thanks, will change it to oc4pre